homeworks/oleksandr.khodakivskyi_Moo0oony/GamesProject/index.html

```diff
@@ -44,5 +47,31 @@ <h1>About me</h1>
                 </article>                
             </section>
         </main>
+        <footer class="footer">
```

Comment:
Please remove this 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/4e48e26e-c5b6-4480-b994-b8603a0ce273)